application/routes/term.py

```
@@ -35,30 +37,34 @@
             return
 
         yield int_to_bytes(ICtrlStep.Term.LAUNCH_SHELL)
         status, reason = term.launch_shell()
         if status is False:
+            logger.error(f"Term: Failed to launch terminal shell: {reason}")
             abort(403, description=reason)
 
         yield int_to_bytes(ICtrlStep.Term.DONE)
         result = {
             'port': TERMINAL_PORT,
             'term_id': term.id
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)